apps/client/src/features/rundown/event-editor/EventEditorWrapper.tsx

```diff
@@ -0,0 +1,140 @@
+  );
+
+  const handleMultipleSubmits = useCallback(
+    (field: EditorUpdateFields, value: string) => {
+      if (field.startsWith('custom-')) {
+        // const fieldLabel = field.split('custom-')[1];
+        // updateEvent({ id: event?.id, custom: { [fieldLabel]: { value } } });
+      } else {
+        // updateEvent({ id: event?.id, [field]: value });
+      }
+    },
+    [event?.id, updateEvent],
+  );
+
+  if (!event) {
+    return (
+      <div className={style.eventEditor} data-testid='editor-container'>
+        Select an event to edit
+      </div>
+    );
+  }
+
+  const getMultipleEvent = (): OntimeEvent => {
+    const allHaveSameValue = (arr: any[], propertyName: string) => {
+      if (!arr || arr.length <= 0) return false;
+      return arr.every((obj) => JSON.stringify(obj[propertyName]) === JSON.stringify(arr[0][propertyName]));
+    };
+
+    const events: OntimeEvent[] = [];
+    let eventsIds: string = '';
+
+    for (const event of selectedEvents) {
+      const data = rundown[event];
+      if (data && isOntimeEvent(data)) {
+        events.push(data);
+        eventsIds += `${event} `;
+      }
+    }
+
+    const multipleEvents: Partial<OntimeEvent> = {
+      id: eventsIds.trim().split(' ').join(', '),
+      colour: allHaveSameValue(events, 'colour') ? events[0]['colour'] : 'unkown',
+      custom: allHaveSameValue(events, 'custom') ? events[0]['custom'] : undefined,
+      duration: allHaveSameValue(events, 'duration') ? events[0]['duration'] : undefined,
+      endAction: allHaveSameValue(events, 'endAction') ? events[0]['endAction'] : EndAction.Unkown,
+      delay: allHaveSameValue(events, 'delay') ? events[0]['delay'] : undefined,
+      isPublic: allHaveSameValue(events, 'isPublic') ? events[0]['isPublic'] : true, // ??
+      linkStart: allHaveSameValue(events, 'linkStart') ? events[0]['linkStart'] : null,
+      note: allHaveSameValue(events, 'note') ? events[0]['note'] : '', // ??
+      revision: allHaveSameValue(events, 'revision') ? events[0]['revision'] : undefined,
+      skip: allHaveSameValue(events, 'skip') ? events[0]['skip'] : undefined,
+      timeDanger: allHaveSameValue(events, 'timeDanger') ? events[0]['timeDanger'] : undefined,
+      timeEnd: allHaveSameValue(events, 'timeEnd') ? events[0]['timeEnd'] : undefined,
+      timeStart: allHaveSameValue(events, 'timeStart') ? events[0]['timeStart'] : undefined,
+      timeStrategy: allHaveSameValue(events, 'timeStrategy') ? events[0]['timeStrategy'] : TimeStrategy.Unkown,
+      timeWarning: allHaveSameValue(events, 'timeWarning') ? events[0]['timeWarning'] : undefined,
+      timerType: allHaveSameValue(events, 'timerType') ? events[0]['timerType'] : TimerType.Unkown,
+      title: allHaveSameValue(events, 'title') ? events[0]['title'] : '', // ??
+      type: allHaveSameValue(events, 'type') ? events[0]['type'] : SupportedEvent.Event,
+      cue: allHaveSameValue(events, 'cue') ? events[0]['cue'] : 'unkown',
+    };
+
```

Comment:
This is where I left off, trying to figure out what the fallback values should be for when the values are not the same for the selected events.

Reply:
Hey Bianca, nice work

here is an example from another event related software

https://github.com/cpvalente/ontime/assets/19875125/98391149-14bf-4eeb-a49d-c37d04ed19f1

values that are identical on both elements stay visible
and element that differ are symbolized with <Multiple Values>

I think that is pretty clear